crates/burn-train/src/renderer/tui/renderer.rs

```diff
@@ -116,9 +117,15 @@ impl TuiMetricsRenderer {
             interuptor,
             popup: PopupState::Empty,
             previous_panic_hook: Some(previous_panic_hook),
+            manual_quit: false,
         }
     }
 
+    /// Enable manual quit after training.
+    pub fn enable_manual_quit(&mut self) {
```

Comment:
I think I would rename the method and take ownership to return the renderer. That way, we can keep the flow similar to the builder when declaring the renderer, e.g. `TuiMetricsRenderer::new(...).persistent()`

```rust
pub fn persistent(mut self) -> Self {
    self.manual_quit = true;
    self
}
```

the `manual_quit` field could also be renamed but not too critical.